diff --git a/pyproject.toml b/pyproject.toml
index 6d06e5f..8b64b79 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -12,7 +12,5 @@ dependencies = [
     "pyusb",
     "pycryptodome",
     "colorama",
-    "shiboken6", 
-    "pyside6",
     "mock"
 ]
diff --git a/requirements.txt b/requirements.txt
index bff165c..13a568d 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -3,7 +3,5 @@ pyusb >= 1.2.1
 pycryptodome >= 3.15.0
 pycryptodomex
 colorama >= 0.4.4
-shiboken6 >= 6.4.0.1
-pyside6 >= 6.4.0.1
 mock >= 4.0.3
 pyserial >= 3.5
diff --git a/setup.py b/setup.py
index 1ab2d68..fc25f2c 100755
--- a/setup.py
+++ b/setup.py
@@ -23,7 +23,6 @@ setup(
     'pyserial',
     'pycryptodome',
     'pycryptodomex',
-    'PySide6',
     'mock'
     ],
     author='B. Kerler',
